code/modules/research/rdconsole.dm

```diff
@@ -1140,8 +1143,6 @@ Nothing else in the console has ID requirements.
 
 /obj/machinery/computer/rdconsole/robotics
 	name = "Robotics R&D Console"
-	req_access = null
-	req_access_txt = "29"
```

Comment:
Why are you removing robo access from the robo's console? Do you not want them researching?

Reply:
Yes

Reply:
Revert this.

Reply:
No